### DISCUSSION TITLE
When can we strart contributing?

### DISCUSSION THREAD

**Discussion by ahmedwael216** [anyproto/contributors#23] [Q&A] [answered]
Is there a specific date, where we can start contributing?

Another question:
Will AnyType's internal issue tracker be published for us to use, Or contributers will have a separate issue management tool?

**Comment by fuksman** [comment]
UPD: Anytype started to work with contributors on July 23, 2023.

Hey @ahmedwael216,

No specific dates yet, but I think we can start accepting contributions next week already. We planned to work with contributors here in GitHub using its tools. 

Can you please explain why you think access to our internal issue tracker could be helpful? Maybe we can give it some thought.

**Reply by vintprox** [reply]
Internal issue trackers do not seem to communicate well with outside contributors. It is good for dealing with the tasks at hand, but this alone doesn't let people just like me to understand that something's being worked on by collaborators internally and what's the progress.

**Reply by ahmedwael216** [reply]
> but this alone doesn't let people just like me to understand that something's being worked on by collaborators internally and what's the progress.

And makes sharing information between the team and the community weaker.

I am asking to publish the internal issue tracker to truly help the team with the community's *already available* knowledge.

**Comment by hit-lacus** [comment]
I think issue tracker is a task database, it will help different level outside contributors to find suitable task to complete. Besides, it will help outside contributors to learn detail of your code.

**Reply by ahmedwael216** [reply]
Totally agree on that.
Plus on the AnyType's team side they might find someone in the community helping them in debugging, fixing bugs or adding new features.

> After all AnyType is built on a lot of very famous frameworks, so most bugs and best ways to solve problems are already known to members of the community.

**Reply by hit-lacus** [reply]
On one hand, issue tracker is not separable from source code. Commit message is not clear to explain everything.

On the another hand, I do think open access to Anytype's internal issue tracker is not a easy task because there will be a lot of problem to be considered(like details of commercial decision  should not open to contributors).

So, there should be two issue tracker systems, but who and how do Anytype sync information between two systems, I wonder how Anytype team deal with this issue.

Thx.

**Comment by fuksman** [comment, marked as answer]
@ahmedwael216 @vintprox @hit-lacus hey, just notifying you that we started working with contributions today. Not in all repositories yet, but it's just a matter of a few commits; I can prioritize repositories if you want to contribute to something specific.
